addons/web/static/src/core/dropdown/dropdown.js

```diff
@@ -275,6 +275,11 @@ export class Dropdown extends Component {
         if (this.ui.activeElement !== this.myActiveEl) {
             return;
         }
+
+        if (ev.target.closest(".bootstrap-datetimepicker-widget")) {
```

Comment:
problem if bootstrap date picker does not come from a sub component of the dropdown???

Reply:
Normally, opening a bootstrap date picker would come from a click, so the dropdown would already be closed.
This may be a problem if a bootstrap date picker is opened in an other manner, but I don't see any case of that.